src/cloud/project/project-webint-snap.md

```diff
@@ -117,6 +117,10 @@ To restore a snapshot using the Magento CLI:
 ## Dump your database {#db-dump}
 
 You can create a copy of your database using [`magento/ece-tools`]({{ site.baseurl }}/cloud/reference/cloud-composer.html#ece-tools).
+By default, the command creates backups using all the connections to the databases that are available through the environment configuration. That is, if split databases are used, by default backup copies will be created for these databases
```

Comment:
```suggestion
You can create a copy of your database using the [`magento/ece-tools`]({{ site.baseurl }}/cloud/reference/cloud-composer.html#ece-tools) `db-dump` command.

By default, this command creates backups for all database connections that are specified in the environment configuration. For example, if you configured your project to use split databases, the `db-dump` operation creates backups for each of the configured databases.
```

Reply:
So you are creating a separate dump file for each database, right?

Reply:
Yes